src/azure-cli/HISTORY.rst

```diff
@@ -25,6 +25,10 @@ Release History
 
 * Fix for issue #5824 - added validation for parameter --rights of authorizationrule commands
 
+**RBAC**
+
+* user: support update command
```

Comment:
Should add the new alias to `--id` from the deprecated option.

Reply:
Done